js/src/oai_wrappers/proxy_wrappers.ts

```diff
@@ -0,0 +1,69 @@
+import { _internalGetGlobalState, getSpanParentObject } from "../logger";
+
+function getProxyBaseUrlDefault({ version }: { version: number }): string {
+  return (
+    process.env["BRAINTRUST_PROXY_URL"] ??
+    `https://braintrustproxy.com/v${version}`
+  );
+}
+
+function getProxyApiKeyDefault(): string | undefined {
+  return process.env["BRAINTRUST_API_KEY"];
+}
```

Comment:
We probably need to check whether process is defined or not. In browser/edge environments, it's not